src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6000,8 +6000,14 @@ checkNullable:
                 Return node
             End If
 
-            Dim featureName = ErrorFactory.ErrorInfo(feature.GetResourceId())
-            Return ReportSyntaxError(node, ERRID.ERR_LanguageVersion, languageVersion.GetErrorName(), featureName)
+            If feature = Feature.InterpolatedStrings Then
+                ' Bug: It is too late in the release cycle to update localized strings.  As a short term measure we will output 
```

Comment:
If this is approved for RTM then there will be an issue filed and the comment will be updated to reflect that.  Holding off on filing the issue until this actually makes it through ship room.